Gameplay/entities/enemies/BossController.cpp

```diff
@@ -157,6 +157,7 @@ void Hachiko::Scripting::BossController::CombatController()
 
 	if (combat_stats->_current_hp <= 0)
 	{
+		level_manager->GoalReached();
```

Comment:
I would do this inside the dead state after a timer or after a button is pressed but for now its ok like this

Reply:
Yep but for now its okey + i dont want to block the scene